src/Item/access.ts

```diff
@@ -37,7 +50,8 @@ export async function hasAccess(
         eth_address
       )
     } else {
-      isManager = isCollectionManager(eth_address, collection)
+      isManager =
+        collection.is_published && isCollectionManager(eth_address, collection)
```

Comment:
same question as [here](https://github.com/decentraland/builder-server/pull/442/files#diff-4f08a39b94a0e600623e156bc99bf67fc48b8cb01d81aacfbb15cf14ccdf35fbR28)

Also, reading here, it seems odd that checking for `is_published` seems to be required for calling `isManager`, if it's a MUST maybe it should be in the function?

Reply:
The problem is that there's an issue with the types here. The method is taking `CollectionAttributes` as the parameter for its type, but `CollectionAttributes` is not correctly typed, it has properties like `managers`, `is_published` and `is_approved` that don't live in the database, making `is_published` possibly null.
We need to fix the collection types to be able to represented a `MergedCollection` containing all of these values and a real `CollectionAttributes` object, but this PR has gotten pretty long already.